content/en/docs/concepts/what-is-opentelemetry.md

```diff
@@ -6,7 +6,7 @@ weight: 10
 ---
 
 Microservices architectures have enabled developers to build and release
-software faster and with greater independence, as they were no longer beholden
+software faster and with greater independence. They are no longer bound
```

Comment:
Unrelated change, not part of https://github.com/open-telemetry/opentelemetry.io/issues/1700